manager-parent/manager-v1/src/main/resources/db/migration/0.1/V0.1.5__create_shards.sql

```diff
@@ -1,5 +1,3 @@
-DROP TABLE IF EXISTS SHARD;
```

Comment:
Is this related with the PR? Btw I agree that it seems strange to drop it everytime

Reply:
I added the `DROP...` to the DDL when I split `manager` into sub-modules as I also create the table in the tests for `manager-core`. Since `manager-v1` *was* including `manager-core` test-jar I had to drop the table otherwise tests failed (as the table had already been created). However the change to the existing DDL file led to Flyway to fail. I have therefore referred the DDL and removed the use of test-jar.